client/entrypoints/amazon-pay-settings/express-checkout-button-preview.js

```diff
@@ -44,9 +44,6 @@ const ExpressCheckoutPreviewComponent = ( { size } ) => {
 		buttonHeight: Math.min( Math.max( height, 40 ), 55 ),
 		paymentMethods: {
 			amazonPay: 'auto',
-			link: 'never',
-			googlePay: 'never',
-			applePay: 'never',
```

Comment:
Removing this, as it is not needed and does not do anything, because only Amazon Pay is in `options.paymentMethodTypes`